### DISCUSSION TITLE
Acknowledging sources in the project

### DISCUSSION THREAD

**Discussion by mhuzzell** [IBAHCM/RPiR#562] [Q&A for project] [answered]
You were very clear that we shouldn't discuss our code for the project with anyone, and I haven't -- but I did ask a question on reddit using a generalised form, and ended up incorporating a (very general-level) suggestion from the answers into my functions. Should I acknowledge this thread in my package, and if so, what is the best way to do that? 

Should we generally be providing a list of sources consulted? I also made pretty heavy use of R for Data Science, for instance.

**Comment by richardreeve** [comment, marked as answer]
If you have sources you'd like to credit, I'd put them in the `README.md` file. The _"don't discuss your code with others"_ in general only applies to other students within the context of university plagiarism rules.

**Reply by mhuzzell** [reply]
Okay, thank you!
